build.gradle

```diff
@@ -73,10 +85,15 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.25'
     compile 'ch.qos.logback:logback-core:1.2.3'
     compile 'ch.qos.logback:logback-classic:1.2.3'
-    compile 'com.datastax.cassandra:cassandra-driver-core:3.6+'
+
+    compile 'com.datastax.cassandra:cassandra-driver-core:3.8+'
```

Comment:
Simulacron has some issues with 3.8 driver. Can we stay with the 3.6? The integration tests failed with 3.8 but downgrading to 3.6 worked.

Reply:
Yeah, we can roll this back.